internal/dag/cache.go

```diff
@@ -59,6 +59,7 @@ type KubernetesCache struct {
 	tlscertificatedelegations map[types.NamespacedName]*contour_api_v1.TLSCertificateDelegation
 	services                  map[types.NamespacedName]*v1.Service
 	namespaces                map[string]*v1.Namespace
+	gatewayclass              *gatewayapi_v1alpha1.GatewayClass
```

Comment:
Might not need to add this to the cache, but for now it's fine until we determine we don't need it here.

Reply:
Yeah, I did not understand the use for it, but I thought it would be best to follow existing patterns.